app/models/clubhouse.rb

```diff
@@ -18,21 +18,25 @@ def self.create_with_owner(owner, properties)
   end
 
   def owners
-    users_of_role(Membership::OWNER_ROLE)
+    role_memberships = memberships.where(
+      role: Membership::OWNER_ROLE
+    )
+    users.where(id: role_memberships.select(:user_id))
   end
 
   def admins
-    users_of_role(Membership::ADMIN_ROLE)
+    role_memberships = memberships.where(
+      'role = ? OR role = ?',
+      Membership::OWNER_ROLE,
+      Membership::ADMIN_ROLE
+    )
```

Comment:
Just a nitpick. You can re-write this to be:
```suggestion
    role_memberships = memberships.where(role: [Membership::OWNER_ROLE, Membership::ADMIN_ROLE])
```